src/args/commit_state.rs

```diff
@@ -2,7 +2,8 @@ use borsh::{BorshDeserialize, BorshSerialize};
 
 #[derive(Default, Debug, BorshSerialize, BorshDeserialize)]
 pub struct CommitStateArgs {
-    /// The ephemeral slot at which the account data is committed
+    /// "Nonce" of an account. Updates are submitted historically and nonce incremented by 1
+    /// Deprecated: The ephemeral slot at which the account data is committed
     pub slot: u64,
```

Comment:
rename to Nonce

Reply:
Ok, renamed in this file, also in CommitRecord and other